tests.js

```diff
@@ -5095,6 +5179,151 @@ window.Specs = {
 		}
 	},
 
+	"predefined-counter-styles": {
```

Comment:
As it's actually a WG Note meant as help for authors and browsers are not required to implement this, it should probably be removed.